sitback-furniture/src/screens/page-not-found/PageNotFound.js

```diff
@@ -0,0 +1,11 @@
+
+import React from 'react';
+
+const PageNotFound = () => {
+    <div>
+        <h1>404 Error</h1>
```

Comment:
could you let me know if we can have two h1 tags in document

Reply:
the other h1 was added by mistake changed it to other h tag